crates/metadata/lib.rs

```diff
@@ -300,12 +300,19 @@ impl std::str::FromStr for Metadata {
             }
         }
 
-        let table = manifest
+        let plane_table = manifest
```

Comment:
```suggestion
        let plain_table = manifest
```

Reply:
@jyn514 Thanks for your review!! I will fix this